src/feedback/trulens/feedback/groundtruth.py

```diff
@@ -24,6 +21,17 @@
 ):
     import evaluate
 
+with OptionalImports(
+    format_import_errors("scipy", purpose="using certain metrics")
+):
+    import scipy.stats as stats
+
+with OptionalImports(
+    format_import_errors("sklearn", purpose="using certain metrics")
```

Comment:
I believe "scikit-learn" is the correct package name while "sklearn" is the module for importing. Fixing this to mention the package instead of module.